gitax/Visual Studio Projects/C Sharp Projects/GTXLibGit2Sharp/GTXRepo.cs

```diff
@@ -20,8 +20,8 @@ public static class GTXRepo
         /// <param name="repoPath">the repository main path</param>
         public static void Init(string repoPath)
         {
-            if (!Repository.IsValid(repoPath))
-                Repository.Init(repoPath, false);
+            if (!Repository.IsValid(repoPath))            
+             Global.error(repoPath  + " is not a valid git repo");            
```

Comment:
I don't have a strong opinion on this. I think a button is a good idea but not required perhaps. Turning a folder into a repo is probably something most people who use git know how to do?
Yes, sync for me is the main problem I want to get working. :)